app/apps/client/gameCenter/gameContainer.js

```diff
@@ -0,0 +1,65 @@
+import { Template } from 'meteor/templating';
+
+import { modal } from '../../../ui-utils';
```

Comment:
always try to import at least to the `client` or `server` level, it makes the autocomplete happier =)
```suggestion
import { modal } from '../../../ui-utils/client';
```